packages/components/src/index.ts

```diff
@@ -581,6 +584,7 @@ import {
     getProjectPath,
     hasModule,
     hasPremiumModule,
+    isAssayDesignExportEnabled,
```

Comment:
I don't see this used in the app side, so it likely doesn't need to be exported here.

Reply:
Yes, true. That was a transitional export.